packages/blockchain-api/config/config.integration.env

```diff
@@ -0,0 +1,6 @@
+BLOCKSCOUT_API=https://integration-blockscout.celo-testnet.org/api
```

Comment:
What is this file for? blockchain-api gets these values from the env variables, which get set automatically by gcloud from the values in the yaml files